jenkins.go

```diff
@@ -302,6 +302,18 @@ func (j *Jenkins) GetJob(id string) (*Job, error) {
 	return nil, errors.New(strconv.Itoa(status))
 }
 
+func (j *Jenkins) GetSubJob(parentId string, childId string) (*Job, error) {
+	job := Job{Jenkins: j, Raw: new(jobResponse), Base: "/job/" + parentId + "/job/" + childId}
```

Comment:
Do we need to validate the ID's at all? What if they are `""` or something? Maybe 

```
if len(parentId) < 1 {
    return nil, fmt.Errorf("Invalid ID")
}
```